### DISCUSSION TITLE
Security Onion installation error

### DISCUSSION THREAD

**Discussion by ZombCry** [Security-Onion-Solutions/securityonion#13065] [2.4] [answered]
### Version

Other (please provide detail below)

### Installation Method

Security Onion ISO image

### Description

installation

### Installation Type

Standalone

### Location

other (please provide detail below)

### Hardware Specs

Meets minimum requirements

### CPU

Ryzen 5 5600G

### RAM

8 gb

### Storage for /

200 gb

### Storage for /nsm

200 gb

### Network Traffic Collection

other (please provide detail below)

### Network Traffic Speeds

Less than 1Gbps

### Status

No, one or more services are failed (please provide detail below)

### Salt Status

No, there are no failures

### Logs

No, there are no additional clues

### Detail

When trying to install the iso image in the virtual machine with vmware it tells me that an error occurs, it has already been tried to install on several occasions and different errors appear in another virtual machine it indicates that there is an error with the disk but it was downloaded from the official website
![Imagen de WhatsApp 2024-05-22 a las 19 39 02_992d7a70](https://github.com/Security-Onion-Solutions/securityonion/assets/101489978/45d5abbc-2f9b-4b3a-aeab-b68c6b1892bc)

### Guidelines

- [X] I have read the discussion guidelines at https://github.com/Security-Onion-Solutions/securityonion/discussions/1720 and assert that I have followed the guidelines.

**Comment by reyesj2** [comment, marked as answer]
It looks like you're trying to install Security Onion 2.3 which is EOL. The latest version can be found at https://docs.securityonion.net/en/2.4/download.html
